src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,123 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY = 16;
+    private int size;
+    private Node<K, V>[] table = (Node<K, V>[]) new Node<?,?>[DEFAULT_CAPACITY];
+    private Node<K, V> nullKeyNode;
 
     @Override
     public void put(K key, V value) {
-
+        if (size <= getThreshold()) {
```

Comment:
The condition should be 'size >= getThreshold()' to resize when the threshold is reached or exceeded, not when the size is below the threshold.